content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -1,54 +1,91 @@
 ---
-title: Plugging in existing CA Certificates
-description: Shows how system administrators can configure Istio's CA with an existing root certificate, signing certificate and key.
+title: Plug in CA Certificates
+description: Shows how system administrators can configure Istio's CA with a root certificate, signing certificate and key.
 weight: 80
 keywords: [security,certificates]
 aliases:
     - /docs/tasks/security/plugin-ca-cert/
 test: true
 ---
 
-This task shows how administrators can configure the Istio certificate authority with an existing root certificate, signing certificate and key.
+This task shows how administrators can configure the Istio certificate authority with a root certificate,
+signing certificate and key.
 
 By default, Istio's CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
 Istio's CA can also sign workload certificates using an administrator-specified certificate and key, and with an
-administrator-specified root certificate. This task demonstrates how to plug such certificates and key into Istio's CA.
+administrator-specified root certificate.
 
-## Plugging in existing certificates and key
+The root CA is used by all workloads within the mesh as the root of trust. Each Istio CA uses an intermediate CA
+signing key and certificate, signed by the root CA. When multiple Istio CAs exist within the mesh, this establishes a
+hierarchy of trust among the CAs.
 
-Suppose we want to have Istio's CA use an existing signing (CA) certificate `ca-cert.pem` and key `ca-key.pem`.
-Furthermore, the certificate `ca-cert.pem` is signed by the root certificate `root-cert.pem`.
-We would like to use `root-cert.pem` as the root certificate for Istio workloads.
+{{< image width="80%"
+    link="hierarchical-pki.png"
+    caption="Hierarchical PKI"
```

Comment:
I wouldn't use acronym `PKI` in the caption, unless it appears in (and is expanded) in the body text as well.